code/modules/mob/living/simple_animal/abnormality/he/red_shoes.dm

```diff
@@ -0,0 +1,410 @@
+		maxHealth += (user.maxHealth * 4.5)
+		revive(full_heal = TRUE, admin_revive = FALSE)
+		add_overlay(mutable_appearance('icons/mob/clothing/feet.dmi', "red_shoes", -ABOVE_MOB_LAYER))
+		add_overlay(mutable_appearance('icons/mob/inhands/weapons/ego_righthand.dmi', "sanguine", -ABOVE_MOB_LAYER))
+		BreachEffect(user)
+		return TRUE
+
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/proc/UnPossess(mob/living/carbon/human/user)//called on death() and BreachEffect()
+	if(HAS_AI_CONTROLLER_TYPE(user, /datum/ai_controller/insane/murder/red_possess))
+		QDEL_NULL(user.ai_controller)
+		user.ai_controller = /datum/ai_controller/insane/murder
+		user.InitializeAIController()
+		user.apply_status_effect(/datum/status_effect/panicked_type/murder)
+
+//BreachEffect and combat
+/mob/living/simple_animal/hostile/abnormality/red_shoes/BreachEffect(mob/living/carbon/human/user)
+	if(!(status_flags & GODMODE))
+		return
+	soundloop.stop()
+	for(var/mob/living/carbon/human/H in GLOB.mob_living_list)//stops possessing people, prevents runtimes. Panicked players are ghosted so use mob_living_list
+		UnPossess(H)
+	if(!possessee)
+		name = "Red Shoe"
+		desc = "The Red Shoes’s bloody enameled leather glistens in the light."
+		icon_state = "redshoes_breach"
+		icon_living = "redshoes_breach"
+		damage_coeff = list(BRUTE = 1, RED_DAMAGE = 0.5, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 1, PALE_DAMAGE = 1.5)
+		new /mob/living/simple_animal/hostile/red_shoe(get_turf(src))
+	..()
+	datum_reference.qliphoth_change(-2)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/Found(atom/A)//The solo breach generally sticks together
+	if(istype(A, /mob/living/simple_animal/hostile/red_shoe))
+		var/mob/living/simple_animal/hostile/red_shoe/S = A
+		if(S.stat != DEAD && !S.target && !S.client && faction_check_mob(S))//cannibalized from steel ordeals
+			S.Goto(src,S.move_to_delay,1)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/AttackingTarget()
+	. = ..()
+	if(!ishuman(target))
+		return
+	var/mob/living/carbon/human/H = target
+	if(H.stat >= SOFT_CRIT || H.health < 0)
+		ChopFeet(H)
+
+/mob/living/simple_animal/hostile/abnormality/red_shoes/proc/ChopFeet(mob/living/carbon/human/H)
+	var/obj/item/bodypart/l_foot = H.get_bodypart(BODY_ZONE_L_LEG)//Feet are defined as BODY_ZONE_PRECISE_L_FOOT. Does the dismember proc not affect them?
+	var/obj/item/bodypart/r_foot = H.get_bodypart(BODY_ZONE_R_LEG)
+	if(HAS_TRAIT(H, TRAIT_NODISMEMBER))
+		return
+	if(!l_foot && !r_foot)//stops spamming the sound effect after being applied once
+		return
+	playsound(src, 'sound/abnormalities/redshoes/RedShoes_Kill.ogg', 100, 1)
+	l_foot?.dismember()
+	r_foot?.dismember()
+
+//***Debuff Definition***/
+//Possession
+/datum/status_effect/red_possess
+	id = "redpossess"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 120 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/red_possess
+	var/sanity_damage_applied = FALSE
```

Comment:
Redundant variable now, it's always applied.

Reply:
Fixed